packages/bot/src/utils/firestore.js

```diff
@@ -1,11 +1,12 @@
-import firebase          from 'firebase';
-import                   'firebase/firestore';
-import { reaction }      from 'mobx';
-import { message_types } from '../constants/messages';
+import firebase             from 'firebase';
+import                     'firebase/firestore';
```

Comment:
- Spacing
- Have you checked whether we're importing Firebase correctly? There's a warning in the developer console when we load the page. This also seems to affect Edge users where they can't load the page.

Reply:
@aaron-binary done.